packages/console/src/pages/ApiResources/components/CreateForm/index.module.scss

```diff
@@ -0,0 +1,32 @@
+@use '@/scss/underscore' as _;
+
+.card {
+  padding: _.unit(8);
+  width: 800px;
+}
+
+.headline {
+  display: flex;
+  justify-content: space-between;
+
+  > *:not(:first-child) {
+    margin-left: _.unit(3);
+  }
+
+  > svg {
+    cursor: pointer;
+  }
+}
+
+.form {
+  margin-top: _.unit(8);
+}
+
+.textField {
+  max-width: 554px;
```

Comment:
we can make a mixin for this

Reply:
since it also applies to `packages/console/src/pages/Applications/components/CreateForm/index.module.scss`